tools/crm_resource_runtime.c

```diff
@@ -1984,7 +1984,7 @@ cli_resource_move(pe_resource_t *rsc, const char *rsc_id, const char *host_name,
     /* only ban the previous location if current location != destination location.
      * it is possible to use -M to enforce a location without regard of where the
      * resource is currently located */
-    if(force && (cur_is_dest == FALSE)) {
+    if(force && (cur_is_dest == false)) {
```

Comment:
I find !cur_is_dest to be more readable and less brittle

Reply:
Fixed.  There's likely more places in this file that could be changed.